internal/network/command_list.go

```diff
@@ -0,0 +1,68 @@
+package network
+
+import (
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+func (c *command) newListCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "Display networks in the current environment.",
+		Args:  cobra.NoArgs,
+		RunE:  c.list,
+	}
+
+	pcmd.AddContextFlag(cmd, c.CLICommand)
+	pcmd.AddEnvironmentFlag(cmd, c.AuthenticatedCLICommand)
+	pcmd.AddOutputFlag(cmd)
+
+	return cmd
+}
+
+func (c *command) list(cmd *cobra.Command, _ []string) error {
+	networks, err := c.getNetworks()
+	if err != nil {
+		return err
+	}
+
+	list := output.NewList(cmd)
+	for _, network := range networks {
+		zones := network.Spec.GetZones()
+		activeConnectionTypes := network.Status.GetActiveConnectionTypes().Items
+
+		if output.GetFormat(cmd) == output.Human {
+			list.Add(&humanOut{
+				Id:                    network.GetId(),
+				EnvironmentId:         network.Spec.Environment.GetId(),
+				Name:                  network.Spec.GetDisplayName(),
+				Cloud:                 network.Spec.GetCloud(),
+				Region:                network.Spec.GetRegion(),
+				Cidr:                  network.Spec.GetCidr(),
+				Zones:                 strings.Join(zones, ", "),
+				DnsResolution:         network.Spec.DnsConfig.GetResolution(),
+				Phase:                 network.Status.GetPhase(),
+				ActiveConnectionTypes: strings.Join(activeConnectionTypes, ", "),
+			})
+		} else {
+			list.Add(&serializedOut{
+				Id:                    network.GetId(),
+				EnvironmentId:         network.Spec.Environment.GetId(),
+				Name:                  network.Spec.GetDisplayName(),
+				Cloud:                 network.Spec.GetCloud(),
+				Region:                network.Spec.GetRegion(),
+				Cidr:                  network.Spec.GetCidr(),
+				Zones:                 zones,
+				DnsResolution:         network.Spec.DnsConfig.GetResolution(),
+				Phase:                 network.Status.GetPhase(),
+				ActiveConnectionTypes: activeConnectionTypes,
+			})
+		}
+	}
+	list.Filter([]string{"Id", "EnvironmentId", "Name", "Cloud", "Region", "Cidr", "Zones", "DnsResolution", "Phase", "ActiveConnectionTypes"})
```

Comment:
No filter needed, since it looks like you are choosing to print all 10 fields.
```suggestion
```

Reply:
I will keep this as we are not printing all 10 fields anymore, removing `EnvironmentId` based on the discussion below.

Reply:
Do we still want to print the environment ID in the `describe` command though?

Reply:
@kabraham13 do you think the environment ID would be useful in the `describe` command?

Reply:
@brianstrauch do you have any concerns if we keep the environment ID in the `describe` command?

Reply:
It's fine! A few of our commands include it, but most don't.